### PR TITLE
Mainnet v3 deploy scripts

### PR DESCRIPTION
**Pull request by iulianpascalau**
- mainnet v3 deploy scripts
- added ledger support

### CODE REVIEW

File: multisig/interaction/release-v3/config-setter.cfg [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+  update-config RELAYER_ADDR_6 erd1cdjeyqcxxeqc8jx0r463rywrew6x26kxq6h42duuxaffm0gjvszq5y8ynv
+  update-config RELAYER_ADDR_7 erd175tjdcmxq47rk0qd3n8fr74n9uaxyfjv5xvel5mp9emvxqk2tdzqdhyrqk
+  update-config RELAYER_ADDR_8 erd17t7kr6u4kzm4r0ftjzpzher38yw9z4hzu96p3hhkz9q944xv43wq8fc6rz
+  update-config RELAYER_ADDR_9 erd1ack50m42nta2x0my4kadfl0ylzu60prxgzuhjhdw6jlujjgmk5yq2wvxcr
+  update-config QUORUM 7
+
+  manual-update-config-file
+}
+
+function set-eth-USDC-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:19]: re-checked: wrong `CHAIN_SPECIFIC_TOKEN_TICKER ETHUSDCJ` setting :x:

**Review by valentin-lup — APPROVED** [review]
Validated ETH side MvX keys.
Validated BSC side MvX keys.
Validated the three Oracles pems.

File: multisig/interaction/release-v3/config-setter.cfg [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0xdac17f958d2ee523a2206206994597c13d831ec7
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 20
+  update-config MAX_AMOUNT 50000
+
+  manual-update-config-file
+}
+
+#function set-eth-BUSD-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:64]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+#  update-config ERC20_TOKEN 0x4fabb145d64652a948d72533023f6e7a623c7c53
+#  update-config MINTBURN_WHITELIST true
+#  update-config NATIVE_TOKEN false
+#  update-config FEE_AMOUNT 25
+#  update-config MAX_AMOUNT 50000
+#
+#  manual-update-config-file
+#}
+
+function set-eth-HMT-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:79]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0xd1ba9bac957322d6e8c07a160a3a8da11a0d2867
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 870
+  update-config MAX_AMOUNT 872750
+
+  manual-update-config-file
+}
+
+function set-eth-CGG-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:94]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0x1fe24f25b1cf609b9c4e7e12d802e3640dfa5e43
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 1853
+  update-config MAX_AMOUNT 635000
+
+  manual-update-config-file
+}
+
+function set-eth-INFRA-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:109]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0x013062189dc3dcc99e9cee714c513033b8d99e3c
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 150
+  update-config MAX_AMOUNT 100000
+
+  manual-update-config-file
+}
+
+function set-eth-WBTC-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:124]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0x2260fac5e5542a773aa44fbcfedf7c193bc2c599
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 0.00032055
+  update-config MAX_AMOUNT 2
+
+  manual-update-config-file
+}
+
+function set-eth-WETH-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:139]: re-checked :heavy_check_mark:

**Review by CostinCarabas — APPROVED** [review]

File: multisig/interaction/release-v3/config-setter.cfg [hunk 8]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 0.00711
+  update-config MAX_AMOUNT 28.5
+
+  manual-update-config-file
+}
+
+function set-eth-WSDAI-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:154]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 9]
```diff
@@ -0,0 +1,285 @@
...
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 20
+  update-config MAX_AMOUNT 50000
+
+  manual-update-config-file
+}
+
+### BSC
+
+function set-bsc-contracts-settings() {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:201]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 10]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0x6fc13eace26590b80cccab1ba5d51890577d83b2
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 2640
+  update-config MAX_AMOUNT 2500000
+
+  manual-update-config-file
+}
+
+function set-eth-WDAI-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:184]: re-checked :heavy_check_mark: 
swap positions with UMB

File: multisig/interaction/release-v3/config-setter.cfg [hunk 11]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0x83f20f44975d03b1b09e64809b757c47f942beea
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 17
+  update-config MAX_AMOUNT 50000
+
+  manual-update-config-file
+}
+
+function set-eth-UMB-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:169]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 12]
```diff
@@ -0,0 +1,285 @@
...
+  update-config RELAYER_ADDR_6 erd10adh0mnzv7s7g4hmr0aj4mpwp3mgr3ykq8lgwg2tkd84jukxyyzqh5tf23
+  update-config RELAYER_ADDR_7 erd1lzssh9anhmxg480jmt0rdxt4v43vjnx9wpjg4psq5c68qthc4gzqs2lcqx
+  update-config RELAYER_ADDR_8 erd1yv639w0lkrvdh5vmyjy8n8e23zh9a55upzx2gr8h4ez0vgw09psqr40w8g
+  update-config RELAYER_ADDR_9 erd1sy2ymmvk2l5z6dkfl5gtzrvvwc9g9jwaaq63t3jfeenagne2fdcqqwm3jq
+  update-config QUORUM 7
+
+  manual-update-config-file
+}
+
+function set-bsc-USDC-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:217]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 13]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 1
+  update-config MAX_AMOUNT 50000
+
+  manual-update-config-file
+}
+
+function set-bsc-USDT-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:232]: re-check :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 14]
```diff
@@ -0,0 +1,285 @@
...
+  update-config ERC20_TOKEN 0x55d398326f99059ff775485246999027b3197955
+  update-config MINTBURN_WHITELIST true
+  update-config NATIVE_TOKEN false
+  update-config FEE_AMOUNT 1
+  update-config MAX_AMOUNT 50000
+
+  manual-update-config-file
+}
+
+# function set-bsc-BUSD-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:247]: re-checked :heavy_check_mark:

File: multisig/interaction/release-v3/config-setter.cfg [hunk 15]
```diff
@@ -0,0 +1,285 @@
...
+#   update-config ERC20_TOKEN 0xe9e7cea3dedca5984780bafc599bd69add087d56
+#   update-config MINTBURN_WHITELIST true
+#   update-config NATIVE_TOKEN false
+#   update-config FEE_AMOUNT 5
+#   update-config MAX_AMOUNT 50000
+#
+#   manual-update-config-file
+# }
+
+function set-bsc-TADA-token-settings {
```
Comment (iulianpascalau) [inline multisig/interaction/release-v3/config-setter.cfg:262]: re-checked :heavy_check_mark:

**Review by sasurobert — APPROVED** [review]

**Pull request merged**